### PR TITLE
DO NOT SUBMIT: Launch amp-forms - remove experiments checks.

### PR DESCRIPTION
**Pull request by mkhatib**
DO NOT SUBMIT until confirming all validation rules are in place and deployed.
#3343

### CODE REVIEW

File: extensions/amp-form/0.1/amp-form.js [hunk 1]
```diff
@@ -35,9 +34,6 @@ import {vsyncFor} from '../../../src/vsync';
 import {actionServiceForDoc} from '../../../src/action';
 import {urls} from '../../../src/config';
 
-/** @type {string} */
-const TAG = 'amp-form';
```
Comment (dvoytenko) [inline extensions/amp-form/0.1/amp-form.js]: Please use this constant for service registration.

**Comment by dvoytenko** [comment]
@mkhatib Thanks! Looks good, with few comments:

1. This PR closes #3998 and #3999 right?
2. This is a cleanup PR, not a launch PR. Launch should just be done via flag launch, which means that this PR should not be merged until then. Is that your plan?

**Comment by kul3r4** [comment]
@mkhatib @rudygalfi Is there a planned date for removing amp-form from experimental?
cc @rowan-m

**Comment by mkhatib** [comment]
Closing this for #6224

**Pull request closed without merging**